R/output.R

```diff
@@ -40,6 +40,21 @@
 #' # To replace errors with a default value, use possibly().
 #' list("a", 10, 100) %>%
 #'   map_dbl(possibly(log, NA_real_))
+#'
+#' # For interactive usage, auto_browse() is useful because it automatically
+#' # starts a browesr() in the right place.
+#' f <- function(x) {
+#'   y <- 20
+#'   if (x > 5) {
+#'     stop("!")
+#'   } else {
+#'     x
+#'   }
+#' }
+#' map(1:6, auto_browse(f))
+#'
+#' # Unfortunately you can't use it with primitive functions like log.
```

Comment:
hmm now that primitive functions are wrapped in a closure I think that should work :)